client/components/Navbar.js

```diff
@@ -12,14 +12,15 @@ class Navbar extends React.Component {
     this.state = {
       userAvatar: 'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png',
       user: false,
+      ConditionalHomeLink: '/',
```

Comment:
The user is automatically redirected to /Dashboard from / if they're logged in. We don't need this particular property.